radio/src/gui/colorlcd/radio_setup.cpp

```diff
@@ -632,6 +632,17 @@ void RadioSetupPage::build(FormWindow * window)
   }
 #endif
 
+#if defined(PWR_BUTTON_PRESS) && defined(PCBX10)
+  // Pwr Off If Inactive
+  {
+    auto line = window->newLine(&grid);
+    new StaticText(line, rect_t{}, STR_PWR_OFF_IF_INACTIVE, 0, COLOR_THEME_PRIMARY1);
+    auto edit = new NumberEdit(line, rect_t{}, 0, 250, GET_SET_DEFAULT(g_eeGeneral.pwrOffIfInactive));
```

Comment:
Discrepancy for max value - Companion max is 255. The value is read from radio.yml and displayed as 255 but radio edit limit is 250.

Reply:
Ah yes, well spotted :-)